src/main/java/com/ericsson/ei/services/SubscriptionService.java

```diff
@@ -217,7 +217,9 @@ private String generateQuery(String subscriptionName, String ldapUserName) {
     private boolean doSubscriptionOwnerExist(String subscriptionName) {
         boolean ownerExist = false;
         try {
-            if (!getSubscription(subscriptionName).getLdapUserName().isEmpty()) {
+            Subscription subscription = getSubscription(subscriptionName);
+            String getLdapUserName = subscription.getLdapUserName();
+            if (getLdapUserName != null && getLdapUserName.isEmpty()) {
```

Comment:
It should be !getLdapUserName.isEmpty() to check that it's not empty. Missing an exclamation mark.

Reply:
Done.